src/olympia/access/acl.py

```diff
@@ -131,13 +134,11 @@ def check_static_theme_reviewer(request):
 def is_reviewer(request, addon):
     """Return True if the user is an addons reviewer, or a personas reviewer
     and the addon is a persona."""
-    return (
-        (check_addons_reviewer(request) and
-         addon.type != amo.ADDON_STATICTHEME) or
-        (check_static_theme_reviewer(request) and
-         addon.type == amo.ADDON_STATICTHEME) or
-        (check_personas_reviewer(request) and addon.is_persona())
-    )
+    if addon.type == amo.ADDON_PERSONA:
+        return check_personas_reviewer(request)
```

Comment:
There is a subtle change hidden in there: previously, someone with `Addons:Review` (the legacy review permission) could review personas just as if they had `Personas:Review`. This was tested as such but didn't make sense to me so I changed it. Is it ok @wagnerand ?

Reply:
imho, it never made much sense that non-theme reviewers could access and review themes (I'm not sure how many of them realised it was possible as we didn't show the links/queues to them)